ansible/README.test.md

```diff
@@ -252,6 +252,11 @@ ansible-playbook test_sonic.yml -i {INVENTORY} --limit {DUT_NAME} -e testcase_na
 ```
 - Requires switch connected to a VM set or PTF testbed
 
+#### Sonic CLI Tab test
+```
+ansible-playbook test_sonic.yml -i {INVENTORY} -e testcase_name=sonic_cli_tab -e testbed_name={TESTBED_NAME}
```

Comment:
A minor update.
kindly add the **"--limit"** option to keep which DUT user wants to use in the testbed while running tests

```ansible-playbook test_sonic.yml -i {INVENTORY} --limit {DUT_NAME} -e testcase_name=sonic_cli_tab -e testbed_name={TESTBED_NAME}```